src/static/js/modules/prepare-worksheets/worksheet-config.js

```diff
@@ -223,7 +228,8 @@ this.worksheetDefaults = {
                 "alternative":"Think of all the little things that you are used to calling your landlord to deal with: a cracked window, a broken dishwasher, or a clogged toilet. As a homeowner, you will either have to fix these yourself or call and pay for a professional.",
                 "altText": "",
                 "explanation":"",
-                "deletable": false
+                "deletable": false,
+                "uneditable": true
```

Comment:
this is a quibble but to me it's a lot of negatives and potentially more confusing to have 'uneditable' rather than an 'editable' key. would be more readable to see if it's editable then check if it's not uneditable, for example here https://github.com/cfpb/owning-a-home/pull/357/files#diff-8685b10fc3ea02fd0a2b3942e2505d01R57